src/effects/native/reverbeffect.cpp

```diff
@@ -31,6 +31,7 @@ EffectManifest ReverbEffect::getManifest() {
     time->setControlHint(EffectManifestParameter::CONTROL_KNOB_LINEAR);
     time->setSemanticHint(EffectManifestParameter::SEMANTIC_UNKNOWN);
     time->setUnitsHint(EffectManifestParameter::UNITS_UNKNOWN);
+    time->setDefaultLinkType(EffectManifestParameter::LINK_LINKED);
```

Comment:
Isn't the bandwidth here just referring to a low-pass filter in the input? In this case a low-bandwidth means more filtering and a higher bandwidth means no filtering... According to my ears, the effect (at least of the filter) is much more audible with the knob at zero than it is at max.